registry/python/container.yaml

```diff
@@ -3,17 +3,10 @@ url: https://hub.docker.com/_/python
 maintainer: '@vsoch'
 description: An interpreted, high-level and general-purpose programming language.
 latest:
-  3.9.10: sha256:5ca3e9ca37dce5ac4614bc307e3bf6759cb5f0a4285cad50e05cacb81c9318a5
+  3.10.0a7-alpine: sha256:9b7958e47cd5bd4d092c3b28802493ad1870bce988b2f6ff97f6c81d96fcda80
 tags:
-  3.9.2-alpine: sha256:f046c06388c0721961fe5c9b6184d2f8aeb7eb01b39601babab06cfd975dae01
-  3.9.2-slim: sha256:ce367bb30b8928efb632c369e3bd4a8dbd7905417bd0a245087a82c250e54a24
-  3.9.4-alpine: sha256:419a7502c95b49946fbdd8228b32243597d9e9f191ddfe5468a3b9b3fe64051d
-  3.9.5-alpine: sha256:7dd8962ad2a63403428d652a64d814a5002f1386379355edf5970e40557fe4e6
-  3.9.6-alpine: sha256:954ea8d05e9041d1dd17b69eb13708a60ef8b8bcc76d928beb4d137e2a9ceb30
-  3.9.7: sha256:8771691756bbf5beff80d64fca8f5b12e018352ddd9e30d8cdfef8cc3717b0e6
-  3.9.9: sha256:dd8335df6162579adadd56ff2b9fbd61199da5405c8856c6e34356c13b48cce4
-  3.9.10: sha256:5ca3e9ca37dce5ac4614bc307e3bf6759cb5f0a4285cad50e05cacb81c9318a5
   3.10.0a7-alpine: sha256:9b7958e47cd5bd4d092c3b28802493ad1870bce988b2f6ff97f6c81d96fcda80
+  3.9.10: sha256:3aae21920963df3205fba69826cc07fcf2fad91f9e062add921766b36e89e6e8
```

Comment:
I see you have the `3.9.*` filter. Do you think it'd be possible to configure the updated to get the latest version of each 3.x ?

Reply:
yep will change to `3[.]*`I was trying to pin to just 3 so I would weed out most of the namespace.